### DISCUSSION TITLE
Keda + GKE "Unable to read all metrics" issue

### DISCUSSION THREAD

**Discussion by gokoppa-user** [kedacore/keda#1853] [Q&A / Need Help] [answered]
### Report

We are using Keda 2.2.0 in GKE with k8s 1.18 to scale a Deployment based with the redis trigger. Both the scaled object and deployment are defined in the same namespace. The minimum amount of replicas is set to 0, max to 5. When adding items to the redis list, KEDA scales from 0 -> 1. The HPA does not scale any further as the autoscaler status is "Unable to read all metrics". When the list is empty again, KEDA also scales back from 1 -> 0. How can I best figure out why the HPA is not able to read all metrics?

### Expected Behavior

That the metrics are read and used to orchestrate the scaling.

### Actual Behavior

When adding items to the redis list, KEDA scales from 0 -> 1. The HPA does not scale any further as the autoscaler status is "Unable to read all metrics". 

### Steps to Reproduce the Problem

1. Use Keda 2.2.0
2. GKE with k8s 1.18
3. Try executing a scaling command.

### Logs from KEDA operator

```
example
```

### KEDA Version

2.2.0

### Kubernetes Version

1.18

### Platform

Google Cloud

**Comment by tomkerkhove** [comment]
Can you share the output of your kubectl commands please?

**Comment by gokoppa-user** [comment, marked as answer]
When diving into the documentation we found out that we needed to add the firewall rule, many thanks for replying, awesome technology :-)

**Reply by tomkerkhove** [reply]
No problem at all, happy to help!
